source/_patterns/01-molecules/navigation/drawer-navigation.twig

```diff
@@ -14,7 +14,6 @@
     <div class="c-drawer__nav">
       <div class="c-drawer__nav-primary">
         {% include '@molecules/navigation/primary-navigation.twig' %}
-        <ul class="c-drawer__subnav u-theme--background-color--darker"></ul>
```

Comment:
@eduludi Is it possible to add a class or CSS change that would make this not necessary to remove? Since we have to coordinate with the update of the Wordpress theme, it would be helpful to be able to leave this line in for at least the next month or so to provide some overlap without messing up the existing installations.

Reply:
It wasn't necessary to remove but, as the JS part to make it useful is not longer there, I just did it. 

Now I rolled back that change and only two files (CSS and JS) are being modified by this PR.

Reply:
The problem I'm seeing is that if you don't remove it, the menu is still unusable on the phone, as it slides out a blank drawer... like before, but with nothing in it. Not sure if that can be hidden?